gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,12 +1,6 @@
-#
-#  Copyright OpenSearch Contributors
-#  SPDX-License-Identifier: Apache-2.0
-#
-
```

Comment:
I think we still need to have a license header, please check core repo, they kept it after gradle 8.1.1 upgrade https://github.com/opensearch-project/OpenSearch/blob/main/gradle/wrapper/gradle-wrapper.properties

Reply:
Good catch